### DISCUSSION TITLE
adding NAS directories and other music content | feature3

### DISCUSSION THREAD

**Discussion by ASchmidt84** [MiczFlor/RPi-Jukebox-RFID#2142] [Q&A]
Hi. After installation works i hoped i can add out network discs with music to RPi.
Maybe with the possiblity to scroll throw it by attachted screen.
Is it also possible to add some other services like spotify or radio network station?

Thank you

**Comment by garytube** [comment]
You simply mount your nas (must be smb share) to the audio folder


mount -t cifs //192.0.2.17/yourAudio /home/jukeboxpath/shared/audiofolder
